WhiteDentalClinic/WhiteDentalClinic.Application/Models/CustomerModel/RequestCreateCustomerModel.cs

```diff
@@ -2,16 +2,19 @@
 
 namespace WhiteDentalClinic.Application.Models.Customer
 {
-    public class CreateCustomerRequestModel
+    public class RequestCreateCustomerModel
     {
         public Guid Id = Guid.NewGuid();
-        [StringLength(50, MinimumLength = 3, ErrorMessage = "First Name should be minimum 3 characters and a maximum of 50 characters")]
+        [StringLength(50, MinimumLength = 3, 
```

Comment:
Add enter before each attribute, please. It is easier to read them if they are grouped. 
```
        [StringLength(50, MinimumLength = 3, 
        ErrorMessage = "Last Name should be minimum 3 characters and a maximum of 50 characters")]
        [DataType(DataType.Text)]
        public string LastName { get; set; }
        [Range(16,110, 
        ErrorMessage = "Age cannot be less than 16 and more than 110.")]
        public int Age { get; set; }
        [DataType(DataType.EmailAddress)]
        public string Email { get; set; }
```
versus 
```
        [StringLength(50, MinimumLength = 3, 
        ErrorMessage = "Last Name should be minimum 3 characters and a maximum of 50 characters")]
        [DataType(DataType.Text)]
        public string LastName { get; set; }

        [Range(16,110, 
        ErrorMessage = "Age cannot be less than 16 and more than 110.")]
        public int Age { get; set; }

        [DataType(DataType.EmailAddress)]
        public string Email { get; set; }
```